index.html

```diff
@@ -417,42 +384,57 @@
     };
 
     callback ThingEventListener = void (ConsumedThing thing);
+
+    enum ConsumedThingEvents {
+        "propertychanged",
+        "propertyremoved",
+        "propertyadded",
+        "actionchanged"
```

Comment:
So we'd either have 
1) `propertychanged` (value), `descriptionchanged` (TD) and `actioninvoked`, 
2) or a flat list of 
`valuechanged` (value),
`propertychanged`, `propertyadded`, `propertyremoved`, `actionadded`, `actionchanged`, `actionremoved` (TD),
`actioninvoked`.